src/cmd/run.go

```diff
@@ -12,4 +10,5 @@ var runCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(runCmd)
+	runCmd.PersistentFlags().String("github-token", "", "Github API Token. Overrides environment variable 'GITHUB_API_TOKEN'")
```

Comment:
Still working on getting this to work with env var too but doesn't seem to work when I add the viper bindenv like in root.go.